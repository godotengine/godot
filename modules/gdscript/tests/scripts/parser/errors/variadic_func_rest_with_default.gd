func f(...args = []):
	pass

func test():
	pass
