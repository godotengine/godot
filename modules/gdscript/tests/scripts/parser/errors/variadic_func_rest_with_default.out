GDTEST_PARSER_ERROR
The rest parameter cannot have a default value.
